### ISSUE TITLE
lists?

### ISSUE THREAD

**Issue by contra** [jxnblk/rebass#46]
open to a PR for lists + different list items?

**Comment by vonwao** [comment]
+1 for lists

**Comment by jxnblk** [comment]
I wanted to get lists in, but put them on the back burner. I was thinking of adding some very generic ones:

- `ListSimple` - unstyled list, with an optional inline prop
- `OrderedList`
- `UnorderedList`
- `ListItem`
- Maybe one with borders

Are there any other types of lists you were imagining?

**Comment by contra** [comment]
@jxnblk Do you have a list somewhere of these things? Could get some people to jump in and help knock it out

**Comment by jxnblk** [comment]
I have some rough notes for myself here: https://github.com/jxnblk/rebass/blob/master/docs/NOTES.md – and contributing doc started here: https://github.com/jxnblk/rebass/blob/master/.github/CONTRIBUTING.md

I can work on cleaning those up, but I'm open to any sort of useful UI component contributions. Generally, I do want to stay away from anything that requires client-side JS (in case that wasn't obvious).

**Comment by mepheser** [comment]
Hi, we just published a basic `<List>` component here: [indoqa-rebass-components](https://github.com/Indoqa/indoqa-rebass-components) 

As our views are getting nice, clean and self-explanatory using only (rebass) components, we try to avoid any raw html markup at all. So we created an abstraction of Lists too. @jxnblk If our `<List>` fits into rebass core, it should be easy to create a pull request after some modifications (adding tests and comments, adopt code styling). Any feedback is appreciated!
